package.json

```diff
@@ -74,7 +74,6 @@
     "source-map-support": "^0.5.13",
     "sudo-prompt": "^9.1.1",
     "username": "^5.1.0",
-    "vite": "^4.5.1",
```

Comment:
Should we keep vite@5 as a top-level dependency? I think we've been mirroring top-level and package-level deps for all other packages too.

Reply:
I'm not sure if `devDependencies` still needs to be put into the top-level deps 🤔, I just moved `vite` in `plugin-vite` form `dependencies` to `devDependencies`.

If necessary, I am willing to add `vite` to the top-level deps. 🙇